tests/unit/src/test/scala/tests/ImplementationSuite.scala

```diff
@@ -0,0 +1,107 @@
+package tests
+import scala.concurrent.Future
+
+object ImplementationSuite extends BaseSlowSuite("implementation") {
+
+  check(
+    "basic",
+    "src/main/scala/a/Main.scala" ->
+      """|package a
+         |trait Livin@@gBeing
+         |abstract class <<Animal>> extends LivingBeing
+         |class <<Dog>> extends Animal
+         |class <<Cat>> extends Animal
+         |""".stripMargin
+  )
+
+  check(
+    "advanced",
+    "src/main/scala/a/LivingBeing.scala" ->
```

Comment:
I would prefer to keep the multiline string syntax for declaring a file layout
```scala
"""|
   |/src/main/scala/a/LivingBeing.scala
   |object foo
   |/src/main/scala/a/Bar.scala
   |object bar
   |""".stripMargin
```

The large number of `"""`, `->`, commas and other syntax makes it harder to read and evolve the file structure.

Reply:
Fixed it :)